ports/esp32/usb.c

```diff
@@ -41,7 +41,8 @@
 static uint8_t usb_rx_buf[CONFIG_TINYUSB_CDC_RX_BUFSIZE];
 
 static void usb_callback_rx(int itf, cdcacm_event_t *event) {
-    // TODO: what happens if more chars come in during this function, are they lost?
```

Comment:
Might be worth adding a comment (on line 42 just before the function), that it's called from a separate FreeRTOS task (and not an ISR).

Reply:
good idea, done